kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/checksum/MetadataChecksumGenerator.java

```diff
@@ -6,18 +6,25 @@
 
 package io.kroxylicious.kubernetes.operator.checksum;
 
+import java.util.Collection;
+import java.util.Comparator;
 import java.util.Map;
 
 import io.fabric8.kubernetes.api.model.HasMetadata;
 import io.fabric8.kubernetes.api.model.ObjectMeta;
 
+import io.kroxylicious.kubernetes.operator.ResourcesUtil;
+
 public interface MetadataChecksumGenerator {
 
     String REFERENT_CHECKSUM_ANNOTATION = "kroxylicious.io/referent-checksum";
     String CHECKSUM_CONTEXT_KEY = "kroxylicious.io/referent-checksum-generator";
     String NO_CHECKSUM_SPECIFIED = "";
 
     static String checksumFor(HasMetadata... metadataSources) {
```

Comment:
maybe better to move all usages to the new method and drop this, we only have one prod usage left of this passing a single element.